From 738390b4f6ea8979d2c4bf0906dc7b218eb73575 Mon Sep 17 00:00:00 2001
From: Margarita Manterola <marga@kinvolk.io>
Date: Thu, 25 Mar 2021 10:48:34 +0000
Subject: [PATCH 1/2] Add Flatcar specific patch to enable the iscsi module

Flatcar uses its own network module instead of the Dracut one, but the iscsi
module depends on the network. So, in order to enable the iscsi module, we need
to patch the dependency.
---
 modules.d/95iscsi/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/module-setup.sh b/modules.d/95iscsi/module-setup.sh
index 2bea2fce..fc9bf068 100755
--- a/modules.d/95iscsi/module-setup.sh
+++ b/modules.d/95iscsi/module-setup.sh
@@ -152,7 +152,7 @@ install_softiscsi() {
 
 # called by dracut
 depends() {
-    echo network rootfs-block
+    echo flatcar-network rootfs-block
 }
 
 # called by dracut
-- 
2.34.1

